packer/scripts/dump-gcp-credentials.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
```

Comment:
How can [Makefile](https://github.com/aeternity/epoch-machine-image/blob/5d80d9cd8bb7fc5d37920956560e71721e4f0fac/Makefile#L8) work after deletion of this?